KarhooUISDK/CommonUI/AddressView/AddressView.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  AddressView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 16/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+import SwiftUI
+
+/// New Address view. Shows pick up, destination and places in between (if needed). View does not provide interactions by default.
+struct KarhooAddressView: View {
+
+    // MARK: - Nested types
+
+    enum Design {
+        /// Bordered view, with SDKs `background1` color and rounded corners.
+        case bordered
+        /// Bordered view, with SDKs `white` color and rounded corners.
+        case borderedWithWhiteBackground
+        /// Bordereless view with `white` color from SDK palette.
+        case `default`
+
+        var backgroundColor: UIColor {
+            switch self {
+            case .bordered:
+                return KarhooUI.colors.background1
+            case .borderedWithWhiteBackground, .default:
+                return KarhooUI.colors.white
+            }
+        }
+
+        var borderColor: UIColor {
+            switch self {
+            case .bordered, .borderedWithWhiteBackground:
+                return KarhooUI.colors.border
+            case .default:
+                return KarhooUI.colors.white
```

Comment:
Again, let's avoid white. The new color palette has been cherry picked into the feature branch and is available there is needed